code/modules/clothing/suits/hood.dm

```diff
@@ -0,0 +1,68 @@
+//Hoods for winter coats and chaplain hoodie etc
+
+/obj/item/clothing/suit/hooded
+	action_button_name = "Adjust hood"
+	var/obj/item/clothing/head/hood
+	var/hoodtype = /obj/item/clothing/head/winterhood //so the chaplain hoodie or other hoodies can override this
```

Comment:
Can go ahead and add 
`species_fit = list("Vox")
 sprite_sheets = list(
 	"Vox" = 'icons/mob/species/vox/suit.dmi'
 	)`